File: app/src/main/java/com/forwork/app/activities/SignUpActivity.java
package com.forwork.app.activities;

import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;

import com.forwork.app.R;

import butterknife.OnClick;

public class SignUpActivity extends AppCompatActivity {
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_sign_up);
    }

    @OnClick(R.id.ivw_back)
    public void onBackClick() {
        finish();
    }
}

File: app/src/main/java/com/forwork/app/activities/SignUpActivity.kt
package com.forwork.app.activities

import android.os.Bundle
import android.support.v7.app.AppCompatActivity

import com.forwork.app.R

import kotlinx.android.synthetic.main.activity_sign_up.*

class SignUpActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_sign_up)

        configure()
    }

    private fun configure() {
        val email = intent.getStringExtra("email")
        if (!email.isNullOrEmpty()) {
            tie_email.setText(email)
        }
    }
}